helmfile/charts/s3-exporter/templates/configmap.yaml

```
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "s3-exporter.fullname" . }}-env
+  labels:
+    {{- include "s3-exporter.labels" . | nindent 4 }}
+data:
+  AWS_REGION: {{ .Values.s3.region }}
+  S3_EXPORTER_S3_ENDPOINT_URL: {{ .Values.s3.regionEndpoint }}
+  S3_EXPORTER_S3_FORCE_PATH_STYLE: {{ .Values.s3.forcePathStyle | quote }}
```

Can we shorten this down?
```suggestion
  S3_ENDPOINT_URL: {{ .Values.s3.regionEndpoint }}
  S3_FORCE_PATH_STYLE: {{ .Values.s3.forcePathStyle | quote }}
```

---

No they come straight from the [s3-exporter](https://github.com/ribbybibby/s3_exporter#flags). The other option would be to set them as flags, but I think it is better like this.

---

Okay.